src/App.tsx

```diff
@@ -40,6 +45,13 @@ function App(props: AppProps) {
         create={CreateQuestions}
         show={QuestionsShow}
       />
+      <Resource
+        name="tracks"
+        list={TracksList}
+        edit={EditGuesser}
```

Comment:
A minor thing, can you implement the edit functionality from scratch please, as we said in discord the guessers are just there to help us when developing but are not production ready and   thus we need to  implement the functionality from scratch